Department-for-Transport-Disruptions/create-disruptions-data
# Add OperatorName field to Siri SX for consequences of type operator

Testing instructions

- Create new disruptions or update existing disruptions with consequences of type operators. Publish the disruptions
- Pull the Siri SX XML using the API which should contain <OperatorName/> field for consequences with type Operators
